pyaedt/modeler/cad/Primitives.py

```diff
@@ -2735,28 +2738,34 @@ def _check_material(self, matname, defaultmatname):
             Name of the material.
         defaultmatname : str
             Name of the default material to assign if ``metname`` does not exist.
+        threshold : float
+            Threshold conductivity to distinguish dielectric from conductor.
```

Comment:
```suggestion
            Threshold conductivity in S/m to distinguish dielectric from conductor.
```
Should the S/m be s/m?